test/regression/cluster.py

```diff
@@ -37,7 +38,8 @@ def __init__(self, solution_file, cluster_file=None, logger=None, logdir=None):
             else:
                 solution_file = os.path.relpath(os.path.join(
                                            os.path.dirname(__file__),
-                                           '../../k8_cortx_cloud/solution.example.yaml'))
+                                           '../../k8_cortx_cloud/'
```

Comment:
Is this missing a comma separating the two strings?

Reply:
This is ok as is.  When one string follows another they will be concatenated.